docs/code/reference.md

```diff
@@ -256,3 +256,10 @@ Don’t forget to make your reports compelling. It’s up to you to sell the mer
 ## Be Prepared to Iterate {#be-prepared-to-iterate}
 
 Don’t be upset if a developer plainly states that they don’t want to implement your idea. Remember that they have plans too. You might have to go back to the drawing board a bit. Listen to their feedback. Your design might be a little over-engineered, it might conflict with other designs being worked on, or maybe it’s just in conflict with the goals and scope of the app. Remember that you’re in the position of requesting someone to devote their time to something. You’re asking for a favor. Don’t be afraid to argue a position within polite reason, but remember to stay humble.
+
+# Other Considerations {#other-considerations}
+
+Here's some other information to take into account before working on elementary applications.
+
+## Gtk Recurrent "Clicked" Issue
+Elementary applications utilize [Gtk](https://www.gtk.org/) (i.e. A GIMP toolkit for GUIs). Gtk widgets are intended to respond to click events that can be described as "press + release" instead of "press". There's been a recurrent issue (see [here](https://github.com/elementary/granite/issues/39) and [here](https://github.com/elementary/granite/issues/82)) in which elementary projects would activate on "press" which has since been patched.
```

Comment:
`elementary` is never capitalized

Reply:
Oh good to know! Should this be added to `reference.md` and/or `human-interface-guildlines.md`? It might be helpful to know not capitalize elementary even when it's at the beginning of a sentence.

Reply:
This is documented at https://elementary.io/brand

I don't think it needs to make its way into the HIG as well.

Reply:
@lewisgoddard It's already in the HIG as [seen here](https://github.com/elementary/website/blob/62b751006961549634e4a0a15d52c5013af130fa/docs/human-interface-guidelines.md#notesexceptions-notes-exceptions)
```
Proper nouns should always be capitalized properly; this means that, for example, Google should always be shown as "Google," elementary should always be shown as "elementary," and MPEG should always be shown as "MPEG."
```
I was suggesting adding a caveat that said something like "elementary should always be shown as elementary, even at the beginning of sentences". But good call on referencing the lowercase mention in https://elementary.io/brand! I'll definitely reference that in the future.

Reply:
That would be a good idea, but that sentence is getting really wordy as it is. I would probably break it up to something like:

> Proper nouns should always be capitalized properly; this means that, for example, Google should always be shown as "Google," and MPEG should always be shown as "MPEG." One major exception is "elementary." It should follow the [brand guidelines](https://elementary.io/brand) and always appear lowercase, even at the start of a sentence.

Reply:
Good call! I like your phrasing better.